src/pages/blog/starlingx-dual-stack-support.md

```diff
@@ -0,0 +1,267 @@
+---
+templateKey: blog-post
+title: Dual-Stack (IPv4/IPv6) Support in Starlingx
+author: Andre Kantek
+date: 2024-08-12T01:32:05.627Z
+category:
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+
+StarlingX, a cloud platform designed for edge computing, now supports dual-stack functionality on its platform networks (OAM, management, cluster-host, cluster-pod, cluster-service, admin, storage, and multicast), enabling it to operate with both IPv4 and IPv6 L3 protocols. This enhanced capability offers greater flexibility and scalability for network deployments. Each platform network can be associated to two pools (one IPv4 and another IPv6). The order it is done defines the primary and secondary pool for that network.
```

Comment:
It would be great to keep these first two paragraphs as a higher level introduction, which focuses on an overview of what the feature is and what it gives to users, why they should care about it. Here you can also add a note that this is a new feature in 10.0, and the rest of the article will focus on how to configure it in Kubernetes.

I would move the second half of this section over into the section that describes how to configure dual-stack in StarlingX 10.0:
"Each platform network can be associated to two pools (one IPv4 and another IPv6). The order it is done defines the primary and secondary pool for that network.

The Kubernetes network can also receive dual-stack properties, allowing the pods to use IPv4 and/or IPv6 on its operations. It becomes available when the OAM, cluster-host, cluster-pod, and cluster-service networks are configured as dual-stack."

Reply:
done